hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx

```diff
@@ -120,40 +120,43 @@ const renderDatanodeOpState = (opState: DatanodeOpState) => {
 };
 
 const COLUMNS = [
+  {
+    title: 'Hostname',
+    dataIndex: 'hostname',
+    key: 'hostname',
+    isVisible: true,
+    isSearchable: true,
+    sorter: (a: IDatanode, b: IDatanode) => a.hostname.localeCompare(b.hostname),
+    defaultSortOrder: 'ascend' as const,
+    fixed: 'left'
+  },
   {
     title: 'State',
     dataIndex: 'state',
     key: 'state',
     isVisible: true,
+    isSearchable: true,
     filterMultiple: true,
     filters: DatanodeStateList.map(state => ({text: state, value: state})),
     onFilter: (value: DatanodeState, record: IDatanode) => record.state === value,
     render: (text: DatanodeState) => renderDatanodeState(text),
     sorter: (a: IDatanode, b: IDatanode) => a.state.localeCompare(b.state),
-    fixed: 'left'
+    //fixed: 'left'
```

Comment:
Could you remove this line instead of commenting it out? 
```suggestion
```

Reply:
Completed Changes as Suggested by Zita